campaigns/tests.py

```diff
@@ -0,0 +1,19 @@
+from django.test import TestCase
+from drip.models import Drip
+from campaigns.models import Campaign, CampaignDrip
+
+DRIP_AMOUNT = 10
+
+
+class DripsTestCase(TestCase):
```

Comment:
Please test also using a class different from Drip, to test the `GenericForeignKey` part